esp32c3-hal/examples/spi_loopback_dma.rs

```diff
@@ -106,10 +106,11 @@ fn main() -> ! {
 
         let transfer = spi.dma_transfer(send, receive).unwrap();
         // here we could do something else while DMA transfer is in progress
-        let mut i = 0;
+        let mut n = 0;
         // Check is_done until the transfer is almost done (32000 bytes at 100kHz is
         // 2.56 seconds), then move to wait().
-        while !transfer.is_done() && i < 10 {
+        while !transfer.is_done() && n < 10 {
+            println!("not is_done; n: {}", n);
```

Comment:
I guess this `println!` is a leftover from testing

Reply:
It is... I thought maybe others would find it useful output, but I've dropped it.